pyscriptjs/rollup.config.js

```diff
@@ -22,29 +46,30 @@ if (!production) {
 
 export default {
     input: 'src/main.ts',
-    output: [
-        {
-            file: 'build/pyscript.js',
-            format: 'iife',
-            sourcemap: true,
-            inlineDynamicImports: true,
-            name: 'pyscript',
-        },
-        {
-            file: 'build/pyscript.min.js',
-            format: 'iife',
-            sourcemap: true,
-            inlineDynamicImports: true,
-            name: 'pyscript',
-            plugins: [terser()],
-        },
-    ],
+    output: [{ minify: true }, { minify: false }].map(({ minify }) => ({
```

Comment:
Where do I pass the `minify` argument to this?

Reply:
This needs to change because of #1298.